README.md

```diff
@@ -10,22 +10,164 @@ Inside the Arsenal Library you have different options to make amazing and quick
+          {
+              @Override
+              public void onTransitionStart(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionEnd(Transition transition)
+              {
+                  //IN HERE YOU CAN START OTHER ANIMATIONS AFTER THE ACTIVITY FINISHES THE 
+                  // ENTER TRANSITION
+                  AnimationArsenal.circularReveal(myView,
+                          getApplicationContext(), AnimationArsenal.RevealGravity.CENTER);
+              }
+
+              @Override
+              public void onTransitionCancel(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionPause(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionResume(Transition transition)
+              {
+              }
+          }, Fade.IN));
+
+
+  //AS WELL YOU CAN HAVE TOTAL CONTROL OF THE EXIT TRANSITION
+  AnimationArsenal.setExitTransition(getWindow(), AnimationArsenal
+          .getExplodeTransition(new Transition.TransitionListener()
+          {
+              @Override
+              public void onTransitionStart(Transition transition)
+              {
+                  //IN HERE YOU CAN START OTHER ANIMATIONS AT THE BEGINNING OF EXIT
+                  // TRANSITION IN THE ACTIVITY
+                  AnimationArsenal.playAnimationFadeOut(getApplicationContext(), myView, 700,
+                          null);
+              }
+
+              @Override
+              public void onTransitionEnd(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionCancel(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionPause(Transition transition)
+              {
+              }
+
+              @Override
+              public void onTransitionResume(Transition transition)
+              {
+              }
+          }, 700));
+```
+Animation Resource Animations
+
+You can play Android Resource Animations as well as custom made animations from your application project.
+```java
+  //ANDROID RESOURCE EXAMPLE
+  AnimationArsenal.playAnimationScale(myView, 700, 0, 1,
+          new Animation.AnimationListener()
+           {
+              @Override
+               public void onAnimationStart(Animation animation)
+              {
+                  //START OTHER ANIMATIONS IF NEEDED
+              }
+
+              @Override
+              public void onAnimationEnd(Animation animation)
+              {
+              }
+
+              @Override
+              public void onAnimationRepeat(Animation animation)
+              {
+              }
+           });
+
+  //APPLICATION RESOURCE EXAMPLE
+  AnimationArsenal.playAnimationFromResource(getApplicationContext(), myView, R.anim
```

Comment:
Wondering, why don't we return the animation, giving the user the flexibility to play it when he wants to play it ?